types/ldapjs-client/index.d.ts

```diff
@@ -0,0 +1,92 @@
+// Type definitions for ldapjs-client 0.1
+// Project: https://github.com/zont/ldapjs-client#readme
+// Definitions by: Valerio Coltrè <https://github.com/colthreepv>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
+
+declare class Filter {
```

Comment:
Is `Filter` actually a class, or is it an interface?

Reply:
I have never used that.
Blatantly copied from [ldapjs](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/ldapjs/index.d.ts#L354)

Honestly [reading carefully the source code](https://github.com/zont/ldapjs-client/blob/master/src/index.js#L96), I'm going to remove this altogether, and just use `string`

Even if the ldapjs-client library has an [entire directory devoted to support Class-Filters](https://github.com/zont/ldapjs-client/tree/master/src/filters)